paddlehub/tokenizer/tokenizer.py

```diff
@@ -204,14 +208,14 @@ def get_input_ids(text):
             if isinstance(text, str):
                 tokens = self.tokenize(text)
                 ids = self.convert_tokens_to_ids(tokens)
-                return self.convert_tokens_to_ids(tokens)
+                return ids
             elif isinstance(text,
                             (list, tuple)) and len(text) > 0 and isinstance(
-                                text[0], str):
+                text[0], str):
```

Comment:
The code style is not the required by PaddleHub. Please commit your codes on python3.6 and install yapf and pre-commit tools, which will check the code style automatically. For more information, please refer to https://github.com/PaddlePaddle/PaddleHub/blob/release/v1.8/docs/contribution/contri_pr.md